docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -122,16 +122,49 @@ The Swarm Scheduler is now active. Commands such as ``deis destroy`` or
 To monitor Swarm Scheduler operations, watch the logs of the swarm-manager
 component, or spy on Docker events directly on the swarm-manager machine:
 
+Kubernetes Scheduler
+--------------------
+
+.. important::
+
+    The Kubernetes Scheduler is a technology preview and is not recommended for
+    production use.
+
+`Kubernetes`_ orchestration system for docker containers:
+
+    Kubernetes provides APIs to manage, deploy and scale docker containers. Kubernetes deploys docker containers as `PODS`_,
+    unique entity across a cluster. Containers inside a POD share the same namespaces. More information about `PODS`_.
+
+To run kubernetes on CoreOS requires `flannel`_ which provides subnet to each host for use with containers runtimes.
+Deis repository provides a new user-data file to bootstrap CoreOs machines on the cluster with flannel.
+
+To test the Kubernetes Scheduler, first install and start the kubernetes components:
+
+.. code-block:: console
+
+    $deisctl install k8s && deisctl start k8s
+
+Then set the controller's ``schedulerModule`` to "k8s":
+
 .. code-block:: console
 
-    $ deisctl journal swarm-manager
-    $ docker -H 172.17.8.102:2395 events
-    2015-04-30T17:31 172.17.8.100:5000/hungry-variable:v5: (from  node:deis-01) pull
-    2015-04-30T17:31 172.17.8.100:5000/hungry-variable:v5: (from  node:deis-02) pull
-    2015-04-30T17:31 02a570: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-01) create
-    2015-04-30T17:31 02a570: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-01) start
-    2015-04-30T17:31 61e59c: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-02) create
-    2015-04-30T17:31 61e59c: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-02) start
+    $ deisctl config controller set schedulerModule=k8s
+
+The Kubernetes scheduler is now active. Commands such as ``deis destroy`` or
+``deis scale web=9`` will use kubernetes ApiServer to manage app PODS.
+
+For each App deis creates a replication controller which manages application PODS and a kubernetes service which acts as a proxy and routes traffic to the PODS associated with the App. A new release uses rolling deploy mechanism. If you are using
+kubernetes scheduler for Deis you can avoid publisher component.
```

Comment:
I would reword this:
```
For each app, Deis creates a replication controller to manage application PODs and a Kubernetes service
which proxies traffic to those PODs. A new app release will use a "rolling deploy" mechanism.
```
Maybe explain specifically how "rolling deploy" differs from what we do with other schedulers? Also I wouldn't bother mentioning that deis-publisher isn't used in this situation. That's an implementation detail, and we don't want to encourage users to rip publisher out IMHO.